pkgs/development/libraries/gsignond/plugins/sasl.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitLab, pkgconfig, meson, ninja, glib, gsignond, gsasl, check
+, gtk-doc, docbook_xml_dtd_43, docbook_xml_dtd_45, docbook_xsl, glibcLocales, gobjectIntrospection }:
+
+stdenv.mkDerivation rec {
+  name = "gsignond-plugin-sasl-${version}";
+  version = "2017-11-11";
+
+  src = fetchFromGitLab {
+    owner = "accounts-sso";
+    repo = "gsignond-plugin-sasl";
+    rev = "671022f3d941509d64249d8f887f4cb60f2a5aab";
+    sha256 = "0arzwr9hl5r1jw8jb6gka7xmizxgbc5yakf7l122iipd4fsyfx6s";
+  };
+
+  nativeBuildInputs = [
+    check
+    glibcLocales
+    gobjectIntrospection
+    meson
+    ninja
+    pkgconfig
+  ];
+
+  buildInputs = [
+    docbook_xml_dtd_43
+    docbook_xml_dtd_45
+    docbook_xsl
+    glib
+    gsasl
+    gsignond
+    gtk-doc
```

Comment:
gtk-doc and docbook stuff is usually only used during build so it should go to `nativeBuildInputs`.